### PR TITLE
Extra product information within the PoS UI

### PR DESCRIPTION
**Pull request by trj-odoo**
In order to get extra information about a product in PoS UI, a contextual window has been created to display it.

Related task id: 2518884

### CODE REVIEW

File: addons/point_of_sale/static/src/css/popups/product_info_popup.css [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+    .product-info-popup .flex-end {
+        align-items: unset;
+    }
+
+    .product-info-popup .section-financials-body {
+        flex-direction: column;
+    }
+
+    .product-info-popup .section-financials-body table {
```
Comment (pimodoo) [inline addons/point_of_sale/static/src/css/popups/product_info_popup.css:144]: is this a table? :D :D :D

Comment (trj-odoo) [reply addons/point_of_sale/static/src/css/popups/product_info_popup.css:144]: shiiiiiiiiiiiiiiiiiit

File: addons/point_of_sale/models/product.py [hunk 2]
```diff
@@ -36,6 +39,61 @@ def _unlink_except_active_pos_session(self):
             if self.with_context(product_ctx).search_count([('id', 'in', self.ids), ('product_tmpl_id.available_in_pos', '=', True)]):
                 raise UserError(_('You cannot delete a product saleable in point of sale while a session is still opened.'))
 
+    def get_product_info_pos(self, price, quantity, pos_config_id):
+        self.check_access_rights('read')
```
Comment (pimodoo) [inline addons/point_of_sale/models/product.py:43]: not needed, if it traceback it is ok, as users in group pos_users, should have access to products

**Comment by pimodoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**